pkg/linear-pools/contracts/midas-linear-pool/MidasLinearPool.sol

```diff
@@ -18,9 +18,9 @@ pragma experimental ABIEncoderV2;
 import "./interfaces/ICToken.sol";
 
 import "@balancer-labs/v2-pool-utils/contracts/lib/ExternalCallLib.sol";
-import "@balancer-labs/v2-pool-utils/contracts/Version.sol";
 
 import "@balancer-labs/v2-pool-linear/contracts/LinearPool.sol";
+import "@balancer-labs/v2-pool-utils/contracts/Version.sol";
 
 contract MidasLinearPool is LinearPool, Version {
     ICToken private immutable _cToken;
```

Comment:
This is redundant storage. We already have access to `getWrappedToken()`, so we don't need to store the wrapped token again. We should simply cast it whenever we need to use it, like so:

```
ICToken(address(getWrappedToken()))
```